index.html

```diff
@@ -9,10 +9,850 @@
+                                <div class="image">
+                                <img class="cardd-image" src="" alt="">
+                            </div>
+                                <h3 class="item_name">Item</h3>
+                                <div class="price-icon">
+                                <p class="price">price</p>
+                                <i  class="fas fa-shopping-cart ccc" title="cart"></i>
+        
+                            </div>
+        
+        
+                            </div>
+                            <div class="cardd">
+                                    <div class="image">
+                                    <img class="cardd-image" src="" alt="">
+                                </div>
+                                    <h3 class="item_name">Item</h3>
+                                    <div class="price-icon">
+                                    <p class="price">price</p>
+                                    <i  class="fas fa-shopping-cart ccc" title="cart"></i>
+            
+                                </div>
+            
+            
+                                </div>
+                            <div class="cardd">
+                                    <div class="image">
+                                    <img class="cardd-image" src="" alt="">
+                                </div>
+                                    <h3 class="item_name">Item</h3>
+                                    <div class="price-icon">
+                                    <p class="price">price</p>
+                                    <i  class="fas fa-shopping-cart ccc" title="cart"></i>
+            
+                                </div>
+            
+            
+                                </div>
+    
+    
+                        </div>
+
+
+            </div>
+
+        </section>
+        <footer>
+            <div class="first-line">
+                <div class="under-stuff">
+                    <i id="under-stuff-id" class="fas fa-envelope"></i>
+                    <div class='under-stuff-div'>
+                        <h3>EMAIL SUPPORT</h3>
+                        <p>help@tosizon.com</p>
+                    </div>
+                </div>
+                <div class="under-stuff">
+                    <i id="under-stuff-id" class="fas fa-phone"></i>
+                    <div class='under-stuff-div'>
+                        <h3>PHONE SUPPORT</h3>
+                        <p>+2348103029715</p>
+                    </div>
+                </div>
+                <formc class="under-form" action="#">
+                <input type="text" name="email" placeholder="email address"><button type="submit">Subscribe</button></form>
+
+            </div>
+            <div class="second-line">
+
+            </div>
+            <hr>
+            <p class="copyright">copyright &copy;2021 tosizon.com </p>
+        </footer>
+            
+    </main>
```

Comment:
No way I'm able to review all this in detail, so from here on out try to keep your work specific in a branch and working on a particular feature to close a single issue.